.lintr

```diff
@@ -14,8 +14,11 @@ linters: all_linters(
     undesirable_function_linter(modify_defaults(
       defaults = default_undesirable_functions,
       library = NULL,
-      options = NULL
-    )),
+      options = NULL,
+      message = "use cli::cli_inform()",
+      warning = "use cli::cli_warn()",
+      stop = "use cli::cli_abort()"
+    ), symbol_is_undesirable = FALSE),
```

Comment:
did `message` as a symbol only come up the once? If so I'd rather just `#nolint` that one case. If this was hitting several times, please ignore :)